src/app/campaign/campaign.component.spec.ts

```diff
@@ -2,7 +2,11 @@ import { TestBed, async, ComponentFixture } from '@angular/core/testing';
 import { DebugElement } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
-import { MatIconModule, MatListModule, MatMenuModule, MatProgressSpinnerModule, MatSidenavModule } from '@angular/material';
```

Comment:
There's an automated migration that updates the material imports for better tree shaking.